app/src/main/java/ch/benediktkoeppel/code/droidplane/MindmapNode.java

```diff
@@ -449,4 +448,16 @@ public Integer getNumericId() {
 
         return numericId;
     }
+	
+    /** Depth-first search in the core text of the nodes in this sub-tree. */
+    List<MindmapNode> search(String searchString) {
+        var res = new ArrayList<MindmapNode>();
+        if (this.text.toUpperCase().contains(searchString.toUpperCase())) {
```

Comment:
I agree most freeplane users are geeks, but regex in a mobile app is maybe too much, no? 
In any case, I'd put it in a separate feature request because it also needs a bit of UI (check-box) to enable it or not.

Reply:
I agree to make this first search feature "K.I.S.S." = Keep it simple, stupid! ; no need to include RegEx search yet.